configure.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+git submodule foreach 'echo "export VERSION_${name//-/_}=$(git describe --tags)" >> ../.env'
+source .env
+envsubst < mkdocs.yml.in > mkdocs.yml
```

Comment:
This script is my hacky approach to make the version tags work.

It fetches the version of each submodule and substitutes them into the `mkdocs.yaml.in` template.

I'm not proud of it but it kind of works...